core-services/prow/02_config/_plugins.yaml

```diff
@@ -2136,6 +2136,30 @@ label:
     - allowed_teams:
       - openshift-patch-managers
       label: backport-risk-assessed
+    - allowed_teams:
+      - openshift-patch-managers
+      label: staff-eng-approved
+    - allowed_teams:
+      - openshift-patch-managers
+      label: jira/valid-bug
+    - allowed_teams:
+      - openshift-patch-managers
+      label: bugzilla/valid-bug
+    - allowed_teams:
+      - openshift-staff-engineers
+      label: cherry-pick-approved
+    - allowed_teams:
```

Comment:
The allowed_teams field is a list, so instead of duplicating each entry you want to do something like

```
- allowed_teams:
  - openshift-staff-engineers
  - openshift-patch-managers
  label: backport-risk-assessed
```

Reply:
if we were going to keep the patch manager team around i'd fix it, but since i'm going to delete the patch manager entries as soon as https://issues.redhat.com/browse/DPP-11105 is done, does it harm anything if i just leave it as is for now?

Reply:
(not sure if the way i have it will break because of the duplicated entries, or is just more verbose than necessary)

Reply:
Yeah, I worry the code that reads the config isn't smart enough to merge the entries and we'll end up with only the last one in the list taking effect. But, I don't know.

Reply:
ok, i'll fix it up

Reply:
@dhellmann fixed